olive/passes/onnx/transformer_optimization.py

```diff
@@ -202,3 +226,36 @@ def _run_for_config(
 
         # save the model to the output path and return the model
         return model_proto_to_olive_model(optimizer.model, output_model_path, config)
+
+    @staticmethod
+    def _replace_mha_with_gqa(model: "OnnxModel", past_seq_len: str = "past_sequence_length", kv_num_heads: int = 0):
```

Comment:
Is there a reason we cannot use `onnxruntime.transformers.convergeneration.replace_mha_with_gqa` directly? Do we want this option to still be usable for older versions of onnxruntime on the host?

Reply:
Yes, old version of ort do not contain the corresponding OPs, so that is not the problem. 

The reason I was thinking: the scripts under convert generation is like temp-solutions, also as this part of logics is quite simple, I directly copy it here.